aiida_quantumespresso/parsers/parse_xml/pw/parse.py

```diff
@@ -167,10 +168,10 @@ def parse_pw_xml_post_6_2(xml):
     inversion_symmetry = False
 
     # See also PW/src/setup.f90
-    nsym = outputs['symmetries']['nsym']  # crystal symmetries
-    nrot = outputs['symmetries']['nrot']  # lattice symmetries
+    nsym = outputs.get('symmetries', {}).get('nsym', None)  # crystal symmetries
```

Comment:
Is this optional only for CP?
BTW, if this function is reused by CP, we should move this function to a top-level folder (i.e. not leave it inside `parse_xml/pw/parse.py` but move it simply to `parse_xml/parse.py`).

Reply:
see 5fbc3ef5 major refactoring